static/styles/settings.scss

```diff
@@ -142,11 +142,14 @@ label {
     box-shadow: 0px 0px 10px hsla(0, 0%, 0%, 0.1);
 }
 
-#change_email_modal,
-#change_full_name_modal {
+#change_email_modal {
     width: 460px;
 }
 
+#change_full_name_modal {
+    width: 480px;
+}
```

Comment:
the alert message requires a long modal width. We can change it to `auto` though.